packages/react/src/components/FlyoutMenu/FlyoutMenu.jsx

```diff
@@ -126,7 +59,7 @@ const DefaultFooter = ({ setIsOpen, onCancel, onApply, i18n }) => (
     <Button
       className={`${iotPrefix}--flyout-menu__submit`}
       aria-label={i18n.applyButtonText}
-      testId="flyout-menu-apply"
+      data-testid="flyout-menu-apply"
```

Comment:
There is a change in https://github.com/carbon-design-system/carbon-addons-iot-react/pull/2042 Which will effect this PR.

i.e. You should use `testID` instead of `data-testid` for PAL `Button`component.

Reply:
Thanks, @herleraja. I've updated it in the newest push. I'm still learning when/where to use `data-testid`, `testID`, or `testId`. Just don't know carbon well enough yet. I'll get there!